service/src/test/java/util/PaymentRequestHelperUtil.java

```diff
@@ -0,0 +1,255 @@
+import com.commercetools.pspadapter.payone.util.PayoneHash;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.neovisionaries.i18n.CountryCode;
+import io.sphere.sdk.carts.Cart;
+import io.sphere.sdk.carts.CartDraft;
+import io.sphere.sdk.carts.CartDraftBuilder;
+import io.sphere.sdk.carts.commands.CartCreateCommand;
+import io.sphere.sdk.carts.commands.CartDeleteCommand;
+import io.sphere.sdk.carts.commands.CartUpdateCommand;
+import io.sphere.sdk.carts.commands.updateactions.AddPayment;
+import io.sphere.sdk.carts.commands.updateactions.SetBillingAddress;
+import io.sphere.sdk.carts.commands.updateactions.SetShippingAddress;
+import io.sphere.sdk.carts.queries.CartQuery;
+import io.sphere.sdk.client.BlockingSphereClient;
+import io.sphere.sdk.client.SphereClientFactory;
+import io.sphere.sdk.models.Address;
+import io.sphere.sdk.models.Versioned;
+import io.sphere.sdk.payments.Payment;
+import io.sphere.sdk.payments.PaymentDraft;
+import io.sphere.sdk.payments.PaymentDraftBuilder;
+import io.sphere.sdk.payments.PaymentMethodInfoBuilder;
+import io.sphere.sdk.payments.TransactionDraftBuilder;
+import io.sphere.sdk.payments.TransactionState;
+import io.sphere.sdk.payments.TransactionType;
+import io.sphere.sdk.payments.commands.PaymentCreateCommand;
+import io.sphere.sdk.payments.commands.PaymentDeleteCommand;
+import io.sphere.sdk.payments.commands.PaymentUpdateCommand;
+import io.sphere.sdk.payments.commands.updateactions.AddTransaction;
+import io.sphere.sdk.payments.queries.PaymentByIdGet;
+import io.sphere.sdk.payments.queries.PaymentQuery;
+import io.sphere.sdk.queries.PagedQueryResult;
+import io.sphere.sdk.queries.QueryPredicate;
+import io.sphere.sdk.types.CustomFieldsDraft;
+import io.sphere.sdk.utils.MoneyImpl;
+
+import javax.annotation.Nonnull;
+import javax.money.Monetary;
+import javax.money.MonetaryAmount;
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.util.Locale;
+import java.util.Random;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.commercetools.util.PayoneHttpClientUtil.nameValue;
+import static java.lang.String.format;
+import static java.util.regex.Pattern.CASE_INSENSITIVE;
+import static java.util.regex.Pattern.DOTALL;
+import static util.PropertiesHelperUtil.getCardPan;
+import static util.PropertiesHelperUtil.getClientId;
+import static util.PropertiesHelperUtil.getClientSecret;
+import static util.PropertiesHelperUtil.getPayoneKey;
+import static util.PropertiesHelperUtil.getPayoneMerchantId;
+import static util.PropertiesHelperUtil.getPayonePortalId;
+import static util.PropertiesHelperUtil.getPayoneSubAccId;
+import static util.PropertiesHelperUtil.getProjectKey;
+import static util.PropertiesHelperUtil.getTenant;
+
+public class PaymentRequestHelperUtil {
+
+    public static final Duration CTP_REQUEST_TIMEOUT = Duration.ofMinutes(2);
```

Comment:
Do you need to make all of them public?

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/392/commits/530108314fae7a172c783b70db7bc7b6d82417d6